From 37c939138e2937f192a3d43a0fbabbe265a0d8bd Mon Sep 17 00:00:00 2001
From: Jason Smith <jsmith@habana.ai>
Date: Fri, 30 May 2025 10:04:44 -0400
Subject: [PATCH] [SW-230673] pdr: gen: avoid line length issues

Some of the record handle enums in the agent are quite long, and
result in checkpatch line-length violations.

This is a brute-force fix that simply splits the record handle
assignment across two lines. It does this for all record handles
not just the ones that too long.

Change-Id: Ib2cc32675b3ca102cdbf410dc9a606420c057ebd
Signed-off-by: Jason Smith <jsmith@habana.ai>
---

diff --git a/scripts/g3-gen-pdrs.py b/scripts/g3-gen-pdrs.py
index 5d91b55..4793a97 100755
--- a/scripts/g3-gen-pdrs.py
+++ b/scripts/g3-gen-pdrs.py
@@ -217,7 +217,8 @@
 			output.write(f'''\
 		.aux_names = {{
 			.header = {{
-				.record_handle = PLDM_RH_{suffix},
+				.record_handle =
+					PLDM_RH_{suffix},
 				.header_version = PLDM_PDR_HEADER_VERSION,
 				.type = {pdr.type.name},
 				.record_change_number = 0,
@@ -338,7 +339,8 @@
 			output.write(f'''\
 		.entity_aux_names = {{
 			.header = {{
-				.record_handle = PLDM_RH_{suffix},
+				.record_handle =
+					PLDM_RH_{suffix},
 				.header_version = PLDM_PDR_HEADER_VERSION,
 				.type = {pdr.type.name},
 				.record_change_number = 0,
@@ -474,7 +476,8 @@
 			output.write(f'''\
 		.oem_state_set = {{
 			.header = {{
-				.record_handle = PLDM_RH_{suffix},
+				.record_handle =
+					PLDM_RH_{suffix},
 				.header_version = PLDM_PDR_HEADER_VERSION,
 				.type = PLDM_PDR_TYPE_OEM_STATE_SET,
 				.record_change_number = 0,
@@ -585,7 +588,8 @@
 			output.write(f'''\
 		.oem_unit = {{
 			.header = {{
-				.record_handle = PLDM_RH_{suffix},
+				.record_handle =
+					PLDM_RH_{suffix},
 				.header_version = PLDM_PDR_HEADER_VERSION,
 				.type = PLDM_PDR_TYPE_OEM_UNIT,
 				.record_change_number = 0,
